setup.py

```diff
@@ -55,7 +55,7 @@ def get_extras_require():
         'checking': ['autopep8', 'hacking'],
         'testing': [
             'pytest', 'mock', 'bokeh', 'plotly', 'chainer>=5.0.0', 'xgboost', 'mpi4py', 'lightgbm',
-            'keras', 'mxnet', 'scikit-optimize', 'tensorflow', 'cma'
+            'keras', 'mxnet', 'scikit-optimize', 'tensorflow', 'cma',
```

Comment:
Please remove the trailing comma to eliminate this unnecessary diff.
```suggestion
            'keras', 'mxnet', 'scikit-optimize', 'tensorflow', 'cma'
```

Reply:
Done